### PR TITLE
[ISSUE #22442] move success message from pypi publish to docker image…

### PR DESCRIPTION
**Pull request by maxi297**
… publish

## What
Following first release after https://github.com/airbytehq/airbyte/pull/23501, I saw two problems:
* bumpversion didn't work properly for the Dockerfile
* we should move the "success" Slack message to the publishing of the Docker image. 

## How
Add Dockerfile to commit when bumping version
Step `publish-cdk` only reports to Slack on error now. `publish-docker-image` reports on either success or failure

### CODE REVIEW

File: .github/workflows/publish-cdk-command-manually.yml [hunk 1]
```diff
@@ -194,33 +194,6 @@ jobs:
         env:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
-      - name: Post success to Slack channel dev-connectors-extensibility
```
Comment (clnoll) [inline .github/workflows/publish-cdk-command-manually.yml:197]: Do we not still want to post success to Slack on `publish-cdk`?

Comment (maxi297) [reply .github/workflows/publish-cdk-command-manually.yml:197]: That's a good question. For me now with the added publishing of the Docker image, I felt that a release is not a success until both of these steps were done. 

Example: Given a workflow that is able to push to pypi and not Dockerhub, I probably want just one message and it is that it failed. In the message, there is a link to the GitHub run and I can easily see which step failed. One thing that might be troublesome in that case is that the pipeline currently does not allow to re-run the pipeline without publishing to pypi so I'll have to bump the version again which is a little bit annoying...

**Review by clnoll — APPROVED** [review]
LGTM!

File: .github/workflows/publish-cdk-command-manually.yml (hunk 1 above)
Comment (clnoll) [reply .github/workflows/publish-cdk-command-manually.yml:197]: 👍 makes sense to me.

**Pull request merged**